manifests/controller/cdi-controller-deployment.yaml

```diff
@@ -10,13 +10,16 @@ metadata:
 rules:
 - apiGroups: [""]
   resources: ["persistentvolumeclaims"]
-  verbs: ["get", "list", "watch", "create", "update", "patch"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
```

Comment:
note to self. Remove "delete". The owner references handle cleaning up the PVC for us now.

Reply:
fixed